template/script.sh.erb

```diff
@@ -6,6 +6,7 @@ echo "TIMING: $(date -Iseconds) - Start script"
 echo "original HOME=$HOME"
 echo "user=$(id -nu)"
 export HOME="/shared/home/$(id -nu)"
+export APPTAINER_HOME=$HOME
```

Comment:
Awesome find! @jaguillette Thanks for the detailed write-up, good to see your process in debugging this thorny detail.